src/components/common/buttons/TaskButton.jsx

```diff
@@ -29,4 +30,35 @@ const TaskButton = ({ buttonText, url, eventTag }) => {
   )
 }
 
+export const WalletV2Continue = ({ buttonText, dontShowAgain, onDismiss }) => {
+  const goToWalletV2 = async () => {
+    try {
+      if (dontShowAgain) {
+        fireEvent('migration_denied')
+        await AsyncStorage.setItem('dontShowWelcomeOffer', true)
+        return
+      }
+
+      fireEvent('migration_accepted')
+      openLink('https://good-wallet-v2.vercel.app/en/promo', '_blank')
```

Comment:
link should be an env var, or you can maybe get it from posthog (as a feature flag) so easier to dynamically change

Reply:
made it part of the feature-flag